ops/terraform/services/server/server-regression/iam.tf

```
@@ -326,3 +326,86 @@ resource "aws_iam_role" "glue_trigger" {
     aws_iam_policy.glue.arn
   ]
 }
+
+resource "aws_iam_role" "spice_refresh_role" {
+  name = "bfd-${local.env}-${local.service}-spice-trigger-role"
+  assume_role_policy = jsonencode(
+    {
+      Statement = [
+        {
+          Action = "sts:AssumeRole"
+          Effect = "Allow"
+          Principal = {
+            Service = "lambda.amazonaws.com"
+          }
+        },
+      ]
+      Version = "2012-10-17"
+    }
+  )
+  force_detach_policies = true
+}
+
+resource "aws_iam_policy" "spice_refresh_logs_policy" {
+  name = "bfd-${local.env}-${local.service}-refresh-spice-trigger-logs"
```

```suggestion
  name = "${local.spice_trigger_lambda_name}-logs"
```

nit: Another nit, but the usual convention for any Lambda IAM Policies is to name the policy starting with the name of the Lambda followed by the service the the Policy pertains to. So, this one would be the name of the Lambda suffixed with `-logs`.